homeassistant/components/shelly/binary_sensor.py

```diff
@@ -64,7 +64,10 @@
         icon="mdi:update",
         default_enabled=False,
         path="update/has_update",
-        attributes={"description": "available version:", "path": "update/new_version"},
+        attributes=[
+            {"description": "released  version:", "path": "update/new_version"},
+            {"description": "installed version:", "path": "update/old_version"},
```

Comment:
```suggestion
            {"description": "released_version", "path": "update/new_version"},
            {"description": "installed_version", "path": "update/old_version"},
```
Attribute names should not contain spaces and `:` sign.
![image](https://user-images.githubusercontent.com/478555/99181556-7da76280-272f-11eb-9ef4-4f13e3b87536.png)